tests/test_mesh_from_lattice/test_mesh_from_lattice.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+import os
+import sys
+sys.path.insert(0, os.pardir)
+sys.path.insert(0, os.path.join(os.pardir, 'openmoc'))
+from testing_harness import TestHarness
+from input_set import OblongLatticeGridInput
+import openmoc.process as process
+import openmoc
+
+
+def get_cell_by_name(cells, name):
+    for c in cells:
+        if c.getName() == name:
+            return c
+
+
+class MeshFromLatticeTestHarness(TestHarness):
+    """An eigenvalue calculation with a mesh tally of the rates
+    using the C++ Mesh class."""
```

Comment:
I think you're using the Python Mesh class here

A C++ mesh is openmoc.Mesh

Reply:
Forgot to update this docstring. Thanks.